libraries/testkit/user.py

```diff
@@ -297,7 +299,7 @@ def update_docs(self, num_revs_per_doc=1, retries=False):
         if len(self.cache.keys()) == 0:
             log.warning("Unable to find any docs to update")
 
-        with concurrent.futures.ThreadPoolExecutor(max_workers=settings.MAX_REQUEST_WORKERS) as executor:
+        with concurrent.futures.ThreadPoolExecutor(max_workers=25) as executor:
```

Comment:
Use constant variable rather than the integer

Reply:
done